app/Console/Commands/Environment/AppSettingsCommand.php

```diff
@@ -62,6 +63,18 @@ public function __construct(private Kernel $console)
      */
     public function handle(): int
     {
+        $this->output->comment('Choose a language for your panel.');
+        /*
+        $langDirectory = 'lang';
+        $files = scandir($langDirectory);
+        $languages = array_filter($files, function($item) use ($langDirectory) {
+            return is_dir($langDirectory . '/' . $item);
+        });
+        */
+        $languages = $this->getAvailableLanguages();
```

Comment:
It might be good to filter down to the completed languages, I think the following should be close:

```suggestion
        $languages = collect($this->getAvailableLanguages())->filter($this->isLanguageTranslated(...))->all();
```